com.github.benpocalypse.Huely.yml

```
@@ -0,0 +1,59 @@
+app-id: com.github.benpocalypse.Huely
+
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: '41'
+base: io.elementary.BaseApp
+base-version: juno-20.08
```

```suggestion
base-version: juno-21.08
```

This is the right version for gnome 41